cibyl/plugins/openstack/sources/zuul/deployments/outlines.py

```diff
@@ -84,6 +89,7 @@ def collect_overrides_for(self, variant: Variant) -> dict:
         """
         result = self._default_overrides()
 
+        self._handle_fs_overrides(variant, result)
```

Comment:
Will give access to the 'featureset_overrides' variable from the tripleo library